src/hooks/usePreferredCurrencies.ts

```diff
@@ -0,0 +1,12 @@
+import { useMemo } from 'react';
+import { getPreferredCurrencies } from '../utils/currencies';
+
+export const usePreferredCurrencies = () => {
```

Comment:
preferred is for me a bit confusing, because it assumes these are the currencies that the user prefers/selected. Why not just `useCurrencies`

And will this data eventually come from the BE?

Reply:
I had to find a different name to describe these currencies beside just currencies as they are not an exhaustive list but a subset of currencies, which the user can choose from.
I ended up doing this because we already have a `currencySelector` component somewhere, I think, for `Checkout` page. I named this component `preferredCurrencySelector` and named relevant currencies same way.

I am open to naming suggestions